projects/cashmere-examples/src/lib/navbar-app-switcher/navbar-app-switcher-example.component.html

```diff
@@ -35,20 +35,45 @@
     </hc-navbar-mobile-menu>
 </hc-navbar>
 
-<hc-pop #appSwitcher><hc-app-switcher iconHeight="100"></hc-app-switcher></hc-pop>
+<hc-pop #appSwitcher [showArrow]="false" horizontalAlign="end">
+    <hc-app-switcher iconHeight="50"></hc-app-switcher>
+</hc-pop>
 
-<hc-pop #options>
-    <ul class="list-options">
-        <li><a href="" target="_blank">Help Topics</a></li>
-        <li><a href="" target="_blank">Release Notes</a></li>
-        <li><a href="" target="_blank">About</a></li>
-        <li><a href="https://community.healthcatalyst.com/" target="_blank">Health Catalyst Community</a></li>
-        <li><a href="" target="_blank">Send us your feedback</a></li>
-    </ul>
+<hc-pop #options [autoCloseOnContentClick]="true" [showArrow]="false" horizontalAlign="end">
+    <div hcMenu>
+        <button hcButtonItem>
```

Comment:
Does it make more sense to use `hcMenuItem` instead of `hcButtonItem`?

Reply:
Good point, agree with that change and will do that now.  As you suggested I'll define both selectors so this doesn't become a breaking change